R/get-results.r

```diff
@@ -47,7 +47,8 @@ id_scenarios <- function(directory){
         x <- unlist(strsplit(all.dirs[i], split="/"))
         return(x[length(x)])
     })
-    scenarios <- temp.dirs[grepl("[A-Z0-9-]+-[a-z]+",temp.dirs)]
+    scenarios <- temp.dirs[grepl("[A-Z0-9-]{2}-[a-z]+",temp.dirs)]
```

Comment:
But it doesn't do that, right?

```
> grepl("[A-Z0-9-]{2}-[a-z]+", "HEREISSOME-stuff")
## [1] TRUE
```

For it to do that it would need starter and end characters:

```
> grepl("^[A-Z0-9-]{2}-[a-z]+$", "HEREISSOME-stuff")
[1] FALSE
> grepl("^[A-Z0-9-]{2}-[a-z]+$", "HE-stuff")
[1] TRUE
```

But a regular expression for only 2 characters wouldn't pick up scenarios like "D0-F1-M1-cod", right?

There's some more complicated regex that would allow for a repeating sequence of capital letter, number, and hyphen, but I'm not sure how that would look off the top of my head.